maquetado/scss/detail.scss

```diff
@@ -0,0 +1,58 @@
+%equal-font-heights {
+ 
+  font-family: Roboto;
+  display: flex;
+  align-items:baseline;
+
+}
+
+%equal-color1 {
+
+  color: #828282;
```

Comment:
Use SASS variables for colors. That way you can re-use them for the rest of the app.